src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.CI.yaml

```diff
@@ -2,6 +2,38 @@
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHjnbqe9AmEW1Js10nySybyuAG7Fb5E9EHUgkmqFDv7PxQGajxbkewesj3PCq3IiGvEkAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMLdJMjR0tj3iBnE3zAgEQgDuyIU42lpmTEO6PIHEU1dyNeIx0UiVTdoUqn4nXWcCOsqlRNWM1o7CyByuiATE23j7d9AIOhwEZxCRvew==
 config:
+  heroku:app_id:
+    secure: v1:d1JeftBc68QcCrCH:tQV5P2EU4H/eF7jMjUied4fIcj2Q6d9ujnhvfn5oxJiPbh41oVff6sWQDs9SUl65gQOTWg==
+  heroku:user: "mitx-devops"
+  heroku_app:vars:
+    CYBERSOURCE_SECURE_ACCEPTANCE_URL: "https://testsecureacceptance.cybersource.com/pay"
+    CYBERSOURCE_WSDL_URL: "https://ics2wstest.ic3.com/commerce/1.x/transactionProcessor/CyberSourceTransaction_1.154.wsdl"
+    DIGITAL_CREDENTIALS_SUPPORTED_RUNS: ""
+    ENABLE_ORDER_RECEIPTS: "True"
```

Comment:
This appears to be `true` for all envs

Reply:
If this is still a functioning feature flag, the code for it should be removed. 

I opened an issue: https://github.com/mitodl/mitxpro/issues/2887